pipeline/alert.go

```diff
@@ -963,6 +968,74 @@ type SensuHandler struct {
 	*AlertNode
 }
 
+// Send the alert to Pushover.
+// Register your application with Pushover at
+// https://pushover.net/apps/build to get a
+// Pushover token.
+//
+// Alert Level Mapping:
+// OK - Sends a -2 priority level.
+// Info - Sends a -1 priority level.
+// Warning - Sends a 0 priority level.
+// Critical - Sends a 1 priority level.
+//
+// Example:
+//    [pushover]
+//      enabled = true
+//      token = "9hiWoDOZ9IbmHsOTeST123ABciWTIqXQVFDo63h9"
+//      user_key = "Pushover"
+//
+// Example:
+//    stream
+//         |alert()
+//             .pushover()
+//              .sound('siren')
+//              .user_key('other user')
+//              .device('mydev')
+//              .title('mytitle')
+//              .URL('myurl')
+//              .URLTitle('mytitle')
+//
+// Send alerts to Pushover.
+//
+// tick:property
+func (a *AlertNode) Pushover() *PushoverHandler {
+	pushover := &PushoverHandler{
+		AlertNode: a,
+	}
+	a.PushoverHandlers = append(a.PushoverHandlers, pushover)
+	return pushover
+}
+
+// tick:embedded:AlertNode.Pushover
+type PushoverHandler struct {
+	*AlertNode
+
+	// User/Group key of your user (or you), viewable when logged
+	// into the Pushover dashboard. Often referred to as USER_KEY
+	// in the Pushover documentation.
+	// If empty uses the user from the configuration.
+	UserKey string
```

Comment:
Since the UserKey is sensitive information should we not allow it to be specified in the TICKscript? Meaning we don't want someones TICKscript to have the user key in plain text. This would restrict it so that you can only use the Pushover integration with a single user. But that seems reasonable and expected since you can only configure a single API token anyways.